device/isis/routerbasic.yaml

```diff
@@ -0,0 +1,18 @@
+components:
+  schemas:
+    Isis.Basic:
+      description: |-
+        This contains ISIS router basic properties.
+      type: object
+      properties:
+        ipv4_te_router_id:
+          description: >-
+            IPv4 Traffic Engineering(TE) router id.     
+          type: string
+          format: ipv4
+          default: "1.1.1.1"
```

Comment:
Is it possible to make it default to first ipv4_loopback address if that is configured under devices? Otherwise use 1.1.1.1 address.

Reply:
We discussed.  Updated the comment. "This address should be configured as an IPv4 Loopback address in 'ipv4_loopbacks'
in the Device."